django/contrib/syndication/views.py

```diff
@@ -160,6 +160,7 @@ def get_feed(self, obj, request):
             feed_copyright=self._get_dynamic_attr("feed_copyright", obj),
             feed_guid=self._get_dynamic_attr("feed_guid", obj),
             ttl=self._get_dynamic_attr("ttl", obj),
+            stylesheet=self._get_dynamic_attr("stylesheet", obj),
```

Comment:
Do we want to support having multiple stylesheets?

Reply:
I wanted to keep the API simple because I assume most people only really have a single stylesheet. I pushed a separate commit that adds documentation (and tests) for how to extend the `Feed` class to support multiple stylesheets.

Reply:
Thank you 👍 that's a lovely addition
This is what I'm thinking. 
If we think we might support multiple in the future, we'd want to deprecate `stylesheet` and add `stylesheets`. In that case it's easier to do now.
Otherwise, as you have documented how to do this in your own code, we'd be recommending users that it should be in your own code and that's a decision to not add it into Django.

Reply:
I've added support for multiple stylesheets (I tried to contain it to a single commit so you can see the diff easier), it didn't add much complexity and your argument about future-proofing against a possible deprecation cycle is convincing.

I did opt to support both a single stylesheet and a list of them, which does make the code (and the documentation) a bit more complex, but I thought it was worth it so that we can offer a simpler API when people only want to use a single stylesheet. If you think that tradeoff is not worth it, I'm also happy to make the API list-only.

Reply:
I don't have too strong of an opinion here. I personally would have just supported lists/tuples (a bit like admin inlines) but I don't mind having this. We can get another opinion 👍

Reply:
With the name `stylesheets` I'd expect to pass an iterable, as a user my immediate reaction is to do `stylesheets=["style.css"]`

Reply:
I'd agree with @yuvadm here :) Thanks so much for working on this PR!